src/popup/popup.css

```diff
@@ -5,13 +5,14 @@ body {
 }
 
 .envs {
-  padding: 20px 0 20px 10px;
+  padding: 0;
   background-color: #000;
 }
 
 .envs a {
   color: #fff;
-  padding: 5px 7px;
+  padding: 15px 8px;
+  display: inline-block;;
```

Comment:
The padding that is removed from `.envs` in the same commit was taking care of the sizing the `.envs` container. Without the padding there would be no block level element to force a sizing:

<img width="566" alt="screen shot 2017-01-24 at 14 25 02" src="https://cloud.githubusercontent.com/assets/233676/22250959/f71da7a6-e240-11e6-9527-091abd0c60be.png">

Double semicolon removed.